### DISCUSSION TITLE
Docker compose Debian

### DISCUSSION THREAD

**Discussion by nan0tEch** [PurpleI2P/i2pd#1984] [General]
Trying to use i2pd via docker in a debian 12 installation.  This code is part of a larger working file.
` #-----------

i2pd:
    image: purplei2p/i2pd:latest
    container_name: i2pd
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=AEurope/Brussels
    volumes:
      - /home/USER/persistent/i2pd:/var/lib/i2pd 
    ports:
      - 7070:7070
      - 4444:4444
      - 4447:4447
      - 7656:7656
    restart: unless-stopped
`
When running the command docker-compose up - d i get the following error by pulling the image

` Pulling i2pd (i2pd:)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling i2pd (i2pd:)...
ERROR: pull access denied for i2pd, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
`

i've tried several different image locations but they all give me the same error.
docker.io/i2pd:latest
i2pd

Any help would be welcome

**Comment by r4sas** [comment]
Looks like Docker Hub somehow blocks container download even we are marked as OSS.
In that case you can try Github Container repository: https://github.com/PurpleI2P/i2pd/pkgs/container/i2pd

**Reply by nan0tEch** [reply]
thx, the next day the container started downloading.
